src/transform/adm1.py

```diff
@@ -0,0 +1,62 @@
+import sys
```

Comment:
The only difference between the different transform scripts is the `search` term. You could make that an input variable and would then only need a single transform method.